DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php

```diff
@@ -95,7 +95,7 @@ public function process(ContainerBuilder $container)
 
                 $groupDefaults[$resolvedGroupName]['items'][] = array(
                     'admin'        => $id,
-                    'label'        => '',
+                    'label'        => !empty($attributes['label']) ? $attributes['label'] : '',
```

Comment:
basically yes - as far as I understood, it would only be set by the setLabel() on the admin class itself in line [279](https://github.com/wodka/SonataAdminBundle/blob/issue-3018-admins-sort/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php#L279)